DistributionPackages/Neos.NeosIo/Resources/Private/Fusion/Content/Column.fusion

```diff
@@ -0,0 +1,34 @@
+prototype(Neos.NeosIo:Content.Column) < prototype(Neos.Neos:ContentComponent) {
+    column = Neos.Neos:ContentCollection {
+        nodePath = '.'
+    }
+    className = ${'grid-layout--item neos-contentcollection'}
+
+    moreLink = ${q(node).property('moreLink')}
+    moreLink.@process.convertUris = Neos.Neos:ConvertUris {
+        absolute = true
+    }
+    moreLinkLabel = Neos.Neos:Editable {
+        property = 'moreLinkLabel'
+    }
+
+    buttonColor = ${q(node).property('style')}
+    buttonAlignment = ${q(node).property('alignment')}
+    buttonClass = 'grid-layout--item--more-link'
+    buttonClass.@process.alignment = ${value + (this.buttonAlignment ? ' u-align' + this.buttonAlignment : '' )}
```

Comment:
```suggestion
```

Reply:
Removed processing but the alignment is for the button and not for the whole column :)